corehq/toggles.py

```diff
@@ -497,6 +497,13 @@ def _ensure_valid_randomness(randomness):
     help_link='https://confluence.dimagi.com/display/ccinternal/Shadow+Modules+and+Forms',
 )
 
+V1_SHADOW_MODULES = StaticToggle(
```

Comment:
I'm not quite following what this toggle does. Who would turn it on? The places where you're checking this toggle, I'd hav expected you to check if the current app has any v1 shadow modules.

Reply:
This gives the option to create a V1 shadow app so that QA can test that the old behaviour still works. However, good point about just checking for if the app has v1 shadows in other places.